external/cmsis/arm_common_tables.h

```diff
@@ -0,0 +1,136 @@
+/* ----------------------------------------------------------------------

```

Comment:
Is there any reason to copy-pste CMSIS in here instead of using submodules like for nrfx and nrfxlib?

Reply:
Done. I have used 4.5.0 tag for the submodule.